File: FinApp/Data/Entities/Expense.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace DAL.Entities
{
    public class Expense
    {
        public int Id { get; set; }

        public int AccountId { get; set; }
        public Account Account { get; set; }

        public int ExpenseCategoryId { get; set; }
        public ExpenseCategory ExpenseCategory { get; set; }

        public int TransactionId { get; set; }
        public Transaction Transaction { get; set; }

    }
}

File: FinApp/FinApp/ClientApp/src/app/models/expense.ts
import { Transaction } from './transaction'
import { ExpenseCategory } from './expenseCategory'
import { Account } from './account'


export class Expense {
  id: number;
  expenseCategory: ExpenseCategory;
  account: Account;
  transaction: Transaction;

  constructor() {
    this.expenseCategory = new ExpenseCategory();
    this.transaction = new Transaction();
    this.account = new Account();
  }
}
